docs/anomaly-detection/README.md

```diff
@@ -0,0 +1,50 @@
+---
+sort: 14
+title: Anomaly Detection
+weight: 7
+disableToc: true
+
+menu:
+  docs:
+    parent: 'victoriametrics'
+    weight: 7
+
+aliases:
+- /anomaly-detection.html
+---
+
+# VictoriaMetrics Anomaly Detection
+
+In the dynamic and complex world of system monitoring, VictoriaMetrics Anomaly Detection, being a part of our [Enterprise offering](https://victoriametrics.com/products/enterprise/), stands as a pivotal tool for achieving advanced observability. It empowers SREs and DevOps teams by automating the intricate task of identifying abnormal behavior in time-series data. This powerful feature goes beyond traditional threshold-based alerting, utilizing sophisticated machine learning techniques to not only detect anomalies but also minimize false positives, thus reducing alert fatigue. By providing simplified alerting mechanisms atop of [unified anomaly scores](/anomaly-detection/docs/models/models.html#vmanomaly-output), it enables teams to spot and address potential issues faster, ensuring system reliability and operational efficiency.
+
+## Key Components and Documentation
+Explore the integral components that configure VictoriaMetrics Anomaly Detection:
+* [Get familiar with components](/anomaly-detection/docs)
+    - [Models](/anomaly-detection/docs/models)
+    - [Monitoring](/anomaly-detection/docs/monitoring.html)
+    - [Reader](/anomaly-detection/docs/reader.html)
+    - [Scheduler](/anomaly-detection/docs/scheduler.html)
+    - [Writer](/anomaly-detection/docs/writer.html)
+
+## Practical Guides and Installation
+Kickstart your journey with our guides and installation instructions:
+* [Quick Start](/anomaly-detection/guides/guide-vmanomaly-vmalert.html) - Set up anomaly detection and alerting
+* Installation Methods
+    - [Docker guide](../vmanomaly.md#run-vmanomaly-docker-container)
```

Comment:
Maybe we need to delete vmanomaly section in docs and put everything into "anomaly detection" section here. Together with docker guide as a "Quickstart Vmanomaly with Docker" or smth.